delta/plugins/jira/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/jira/JiraClient.scala

```diff
@@ -146,7 +146,7 @@ object JiraClient {
                             case None                      => IO.raiseError(NoTokenError)
                             case Some(_: AccessToken)      => IO.raiseError(RequestTokenExpected)
                             case Some(RequestToken(value)) =>
-                              IO {
+                              IO.blocking {
```

Comment:
It might be clearer to put only the execute part in blocking, it was confusing initially